README.md

```diff
@@ -19,6 +19,46 @@ const auth = new Auth()
 await auth.login() // prompts the user to login
 
 const accessToken = await auth.getToken() // returns a valid access token
+const userKey = auth.getUserKey() // returns the key used to get the token
+```
+
+#### Send signed request
+
+With the `accessToken` and the `userKey` you should be able to send a signed request to the service provider you are trying to reach.
+
+```ts
+const AuthModule =  require('decentraland-auth-protocol')
+
+let accessToken = await auth.getToken()
+const userKey = auth.getUserKey()
+const fullURL = 'https://theService.com/path/param/?query=something'
+
+// GET request example
+const input = AuthModule.MessageInput.fromHttpRequest("GET", fullURL, null)
+  
+let requestMandatoryHeaders = userKey.makeMessageCredentials(input, accessToken)
+
+const response = await fetch(fullURL, {
+    method: 'get',
+    headers: requestMandatoryHeaders
+  })
+
+// POST request example
+accessToken = await auth.getToken()
+const body = Buffer.from(
```

Comment:
`Buffer` is not available on the browser, is this the only way to send a POST message?

Reply:
Updated readme